tests/test_entrypoint/database/test_parent.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
+
+import pytest
+
+
+@pytest.mark.parametrize("command", ["add", "init"])
+def test_commands_work(command):
+    subprocess.check_call(["bbs_database", command, "--help"])
```

Comment:
For some reason this actually does not work.
```
parent.main([command, "--help"])
```
It raises SystemExit error when launched via `pytest`. However, works completely fine when run with plain `python`.

Reply:
Isn't that expected?

Reply:
Well, your suggestion cannot be used because `argparse` actually calls `sys.exit(0)` after it displays the help message. Which just kills the Python interpreter.
```python
# example.py
import argparse                                                                                                                                                                                       
                                                                                                                                                                                                      
parser = argparse.ArgumentParser()                                                                                                                                                                    
                                                                                                                                                                                                      
args = parser.parse_args(["-h"])                                                                                                                                                                      
print("Unreachable") 
```
```bash
python example.py
```
```bash
usage: example.py [-h]

optional arguments:
  -h, --help  show this help message and exit
```

Logically, `pytest` sees this as a failure. Even despite the fact that we exited with code 0.
```python
# test_example.py
import sys

def test():
    sys.exit(0)
    assert True

```
```bash
pytest test_example.py
```
```bash
...
FAILED test__example.py::test - SystemExit: 0
```